estate/models/estate_property.py

```diff
@@ -0,0 +1,47 @@
+from odoo import models, fields
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Create a new table"
```

Comment:
Description should be a human readable name for you model.
e.g. for model with `_name = 'sale.order'` we have `_description = "Sales Order"`